datafiles/templates/index.html.st

```diff
@@ -33,8 +33,8 @@ $hackagePageHeader()$
 </ul>
 <p>Guidelines for Hackage Packages:</p>
 <ul>
-<li><strong>Packages cannot be deleted</strong>, so you should consider uploading new package versions as <a href="/packages/candidates/upload">package candidates</a> and testing before publishing it to the main index.</li>   
-<li>All curated packages <abbr title="[RFC2119] The word 'should' is intended to denote that there may exist valid reasons in particular circumstances to ignore a particular item, but the full implications MUST be understood and carefully weighed before choosing a different course">should</abbr> follow the <a href="https://pvp.haskell.org/">Package Versioning Policy (PVP)</a>.</li>
+<li><strong>Packages cannot be deleted</strong>, so you should consider uploading new package versions as <a href="/packages/candidates/upload">package candidates</a> and testing before publishing it to the main index.</li>
```

Comment:
"testing before publishing it" should be "testing before publishing them" to agree with the plurals earlier.